dataactvalidator/scripts/load_program_activity.py

```diff
@@ -120,13 +120,14 @@ def load_program_activity_data(base_path):
             data = pd.read_csv(program_activity_file, dtype=str)
         except pd.io.common.EmptyDataError as e:
             log_blank_file()
-            sys.exit(4)  # exit code chosen arbitrarily, to indicate distinct failure states
-
+            if not exit_if_nonlocal(4):  # exit code chosen arbitrarily, to indicate distinct failure states
```

Comment:
We don't really need to make these `if` statements. Just run `exit_if_nonlocal` and then return after it, since it won't ever get to the return if it's not local